### PR TITLE
Tyshchenko/rvat 645 check present change button

### PR DESCRIPTION
**Pull request by TyshchenkoRoma**

### CODE REVIEW

**Review by MaxKovalets — APPROVED** [review]

**Review by JugerJuger — APPROVED** [review]

File: src/test/java/rv/atqc/gttesting/editmyprofiledata/US_EditMyProfileDataWithChrome.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+public class US_EditMyProfileDataWithChrome {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setUpDriverAndPreconditions() {
+        ChromeDriverManager.getInstance().setup();
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editmyprofiledata/US_EditMyProfileDataWithChrome.java]: please use explicit waiter

File: src/test/java/rv/atqc/gttesting/editmyprofiledata/US_EditMyProfileDataWithFirefox.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+public class US_EditMyProfileDataWithFirefox {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setUpDriverAndPreconditions() {
+        FirefoxDriverManager.getInstance().setup();
+        driver = new FirefoxDriver();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editmyprofiledata/US_EditMyProfileDataWithFirefox.java]: please use explicit waiter

**Review by dmatsev — COMMENTED** [review]
Please fix

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please update your US branch

**Review by dmatsev — APPROVED** [review]

**Review by dmatsev — APPROVED** [review]

**Pull request merged**